src/main/java/betterquesting/questing/QuestInstance.java

```diff
@@ -158,6 +161,15 @@ else if (!canSubmit(player))
 					{
 						done++;
 						update = true;
+                        if (hasConsumableTasks && QBConfig.partySyncConsumeQuests) {
+                            synchronized (completeUsers) {
```

Comment:
It just worked. Not sure if it is okay to use _synchronized_ here

Reply:
doesn't really work. consumable quests rewards are not claimable at all with this config

Reply:
Looks like it works correctly on localhost but make rewards not claimable on LAN and Internet clients. I'll try to figure out but I'll appreciate any help :)